app/controllers/variable_lookup_keys_controller.rb

```diff
@@ -0,0 +1,2 @@
+class VariableLookupKeysController < LookupKeysController
```

Comment:
when you have a form_for object, it points to the correct controller, thus, the update action on VarLookupKeys will be called. Basically anything which uses rails' helpers for the object passed to them.